_infra/helm/securebanking-openbanking-uk-fidc-initializer/templates/cronjob-secret.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: kubernetes-client.io/v1
 kind: ExternalSecret
 metadata:
   name: initializer-secret
-  namespace: {{ .Values.environment.namespace }}
 spec:
```

Comment:
This namespace directive will be necessary to deploy the initialiser in each namespace, isn't? so that probably isn't a good idea delete it.

Reply:
The metadata.namespace doesn't need to be specified in each object definition, it will be set by the --namespace flag that argo passes to helm.

For instance, if we look at [configmap.yaml](https://github.com/SecureBankingAccessToolkit/securebanking-openbanking-uk-fidc-initializer/blob/master/_infra/helm/securebanking-openbanking-uk-fidc-initializer/templates/configmap.yaml) , we don't have a metadata.namespace specified but if we check each namespace we see the configmap has been installed.